internal/pkg/term/selector/selector.go

```diff
@@ -427,6 +427,52 @@ func (s *Select) Environment(prompt, help, app string, additionalOpts ...string)
 	return selectedEnvName, nil
 }
 
+// Environments fetches all the environments in an app and prompts the user to select one OR MORE.
+// The List of options decreases as envs are chosen. Chosen envs displayed above with the finalMsg.
+func (s *Select) Environments(prompt, help, app string, finalMsg prompt.Option, additionalOpts ...string) ([]string, error) {
+	envs, err := s.retrieveEnvironments(app)
+	if err != nil {
+		return nil, fmt.Errorf("get environments for app %s from metadata store: %w", app, err)
+	}
+	if len(envs) == 0 {
+		log.Infof("Couldn't find any environments associated with app %s, try initializing one: %s\n",
```

Comment:
Do we log for any selectors in this pkg?
The general idea is that only the `cli` pkg should log while the remaining pkgs are libraries that could be used by our `cli` or any other client!
I don't mind it so much for the `selector` pkg because really it only  makes sense for it to be used by the CLI but wanted to spread the knowledge here 💭

Reply:
Yeah: DeployedService, Job, Workload, Service, Environment, Application, AskCron. :joy:
But that makes sense, good to know.